### PR TITLE
Accept .LAS, .LAZ and .PLY file extensions

### LINKED ISSUE #180: Displaz does not open uppercase .LAS, .LAZ
For whatever reason, displaz can't open a file ending in `.LAS`. It logs `ERROR: Could not load x.LAS`.

Renaming the file to `x.las` resolves the problem.

### PR DESCRIPTION
**Pull request by andyferris**
Previously, lowercase was assumed. Fixes #180 

@c42f There are some other file extensions like .hcloud, .com and .pointdb which I just did the same for... I assume that's good.

### CODE REVIEW

**Comment by nigels-com** [comment]
Looks good to me.

File: src/main.cpp [hunk 1]
```diff
@@ -204,7 +204,7 @@ int main(int argc, char *argv[])
         // Launch the main GUI window in a separate process.
         QString exeName = QCoreApplication::applicationFilePath();
 #       ifdef _WIN32
-        if (exeName.endsWith(".com"))
+        if (exeName.toLower().endsWith(".com"))
```
Comment (c42f) [inline src/main.cpp]: This change isn't required. It's part of an internal hack to make displaz appear to be a dual GUI/console mode program on windows. See https://github.com/c42f/displaz/blob/5e38fa70475c1b5e32caa75ada37ce4f58404d95/src/CMakeLists.txt#L146

Comment (andyferris) [reply src/main.cpp]: removed

**Comment by andyferris** [comment]
Lol, MacOS build is still queued (travis reduced their mac build farm and everyone is waiting).

**Comment by andyferris** [comment]
Bump

**Comment by c42f** [comment]
Looks good to me, cheers.

**Pull request merged**